src/Tests/Eshopworld.Tests.Core.Analyzers.Test/Eshopworld.Tests.Core.Analyzers.Test.csproj

```diff
@@ -13,6 +14,8 @@
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
```

Comment:
unnecessary?

Reply:
looks like it's safe to remove
https://stackoverflow.com/a/24055340